backend/src/db/dbAccessor.ts

```diff
@@ -0,0 +1,38 @@
+import {Database} from "sqlite3";
+
+const sqlite3 = require('sqlite3').verbose();
+import path from 'path';
+
+const accessDB = (functionToExecute: (db: Database) => any, verbose: boolean = false) => {
```

Comment:
Warum returnt functionToExecute any? Da der Rückgabewert bei db.serialize übergeben wird, sollte das doch einen bestimmten Typ haben, ich denke mal () => void ?
https://stackoverflow.com/questions/41949724/how-does-db-serialize-work-in-node-sqlite3

Reply:
Hast du das ausprobiert? Der Typ 'Database' wird doch gar nicht importiert.

Reply:
der Typ Database kommt über sqlite3

Reply:
rückgabewert angepasst